app/views/layouts/darkswarm.html.haml

```diff
@@ -50,4 +33,23 @@
 
           #footer
     %loading
+
+    %script{:src => "https://js.stripe.com/v3/", :type => "text/javascript"}
+    %script{src: "//maps.googleapis.com/maps/api/js?libraries=places,geometry#{ ENV['GOOGLE_MAPS_API_KEY'] ? '&key=' + ENV['GOOGLE_MAPS_API_KEY'] : ''} "}
+    = javascript_include_tag "darkswarm/all"
+    = javascript_include_tag "web/all"
+    = render "layouts/i18n_script"
```

Comment:
we need to make sure on slow page loads the page doesnt load without the translations - this needs testing  in pages with lots of content